content/influxdb/v2.0/reference/flux/stdlib/profiler/_index.md

```diff
@@ -21,6 +21,9 @@ Import the `profiler` package:
 import "profiler"
 ```
 
+Each enabled profiler appends a table to the output stream of tables containing
+data returned by the profiler.
```

Comment:
How would you enable two profilers rather than one? Could you append multiple tables to a stream with multiple profilers?

Reply:
Yes, you can. The examples on the page show how.